packages/react-components/react-positioning/src/usePositioning.ts

```diff
@@ -125,6 +125,8 @@ export function usePositioning(
   useIsomorphicLayoutEffect(() => {
     if (options.target) {
       overrideTargetRef.current = options.target;
+    } else {
+      overrideTargetRef.current = null;
     }
```

Comment:
Isn't this the same as the below one liner ?
```ts
overrideTargetRef.current = options.target;
```

Reply:
updated :)